doc/src/cylc-user-guide/suiterc.tex

```diff
@@ -115,23 +115,32 @@ \subsection{[cylc]}
 negative sign, thereafter following the ISO 8601 standard example notation
 except for fractional digits, which are represented as \lstinline=,ii= for
 \lstinline=hh=, \lstinline=,nn= for \lstinline=mm=, etc. For example, to write
-date/times as week dates with fractional hours, set cycle point format to
-\lstinline=CCYYWwwDThh,iiZ= e.g. \lstinline=1987W041T08,5Z= for 08:30 UTC on
+date-times as week dates with fractional hours, set cycle point format to
+\lstinline=CCYYWwwDThh,iiZ= e.g.\  \lstinline=1987W041T08,5Z= for 08:30 UTC on
 Monday on the fourth ISO week of 1987.
 
 You can also use a subset of the strptime/strftime POSIX standard - supported
-tokens are \lstinline=\%F=, \lstinline=\%H=, \lstinline=\%M=, \lstinline=\%S=,
-\lstinline=\%Y=, \lstinline=\%d=, \lstinline=\%j=, \lstinline=\%m=,
-\lstinline=\%s=, \lstinline=\%z=.
+tokens are \lstinline=%F=, \lstinline=%H=, \lstinline=%M=, \lstinline=%S=,
+\lstinline=%Y=, \lstinline=%d=, \lstinline=%j=, \lstinline=%m=,
+\lstinline=%s=, \lstinline=%z=.
 
-To use the old cylc date-time format (e.g. \lstinline=2014020106= for 06:00
-on the 1st of February 2014), set cycle point format to
-\lstinline=\%Y\%m\%d\%H=.
+The ISO8601 extended date-time format can be used
+(\lstinline=%Y-%m-%dT%H:%M=) but
+note that the `-' and `:' characters end up in job log directory paths.
 
-Please note that using characters like "/" is not allowed, as it will break
-task output files ("/" is a reserved character in POSIX file and directory
-naming). Using ":" is also not allowed, as it is likely to interfere with
-usage of commands like "rsync" when applied to task output files.
```

Comment:
This advice seems to be unnecessary.  You can't use "/", and job log retrieval works fine with `%Y-%m-%dT%H:%M`.